backend/tracim_backend/app_models/contents.py

```diff
@@ -105,34 +107,59 @@ def get_default_status(self) -> ContentStatus:
 # ContentType
 
 
-class ContentType(object):
+class ContentTypeInContext(object):
```

Comment:
same suggestion: ConfigReadyContentType? (not sure, please explain if you think this is not a good idea)

Reply:
same explanation as for app.